docs/source/conf.py

```diff
@@ -30,7 +31,15 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinxcontrib.bibtex", "sphinx.ext.autodoc", "numpydoc"]
```

Comment:
`sphinxcontrib.bibtex` not longer needed?

Reply:
Some docs refer to some weird bib syntax, so I guess it is still needed.